crates/fee/src/lib.rs

```diff
@@ -134,6 +134,9 @@ decl_storage! {
 
         // NOTE: currently there are no collator rewards
         CollatorRewards get(fn collator_rewards) config(): UnsignedFixedPoint<T>;
+
+        /// Percentage of fees generated by nominated collateral that is given to nominators
+        NominationRewards get(fn nomination_rewards) config(): UnsignedFixedPoint<T>;
```

Comment:
Won't the rewards vary by participant?

Reply:
They will vary based on the amount of nominated collateral. However, the way rewards generated by a nomination are split is always the same (`nomination_rewards`% goes to the nominator, `100 - nomination_rewards`% goes to the operator)